### PR TITLE
added php classes

### LINKED ISSUE #272: Add PHP Classes
# Description

Classes for PHP aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/classes.json` to the right PHP directory in `web/thesauruses/`
* Edit the classes.json file to accurately reflect the latest major version of PHP
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by apoorvpal01**
## What GitHub issue does this PR apply to?

Resolves #272 

## What changed and why?

Added JSON formatted information for PHP Classes.

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [x] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/php/classes.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
+{
+    "meta": {
+      "language": "language_id",
+      "language_version": "version.number",
+      "language_name": "Human-Friendly Language Name"
```
Comment (geekygirlsarah) [inline web/thesauruses/php/classes.json]: Can you update these to `php`, whatever major/minor version is the latest (7.4?), and `PHP`?

File: web/thesauruses/php/classes.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        "code": "class ClassName {\n //Class body containing variables and methods \n}"
+      },
+      "abstract_class": {
+        "code": "abstract class ClassName {\n //Class body containing methods (atleast one of which is abstract) \n}"
+      },
+      "interface": {
+        "code": "interface InterfaceName {\n //Interface body containing method declarations \n}"
+      },
+      "read_only_class": {
+        "code": "",
```
Comment (geekygirlsarah) [inline web/thesauruses/php/classes.json]: You can remove all `code` lines for anything that's not implemented.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
Practically there! Make the minor changes, and I'll merge this in!

File: web/thesauruses/php/classes.json (hunk 1 above)
Comment (apoorvpal01) [reply web/thesauruses/php/classes.json]: I have added the language data in a new commit. The current stable version is `8.0`

File: web/thesauruses/php/classes.json (hunk 2 above)
Comment (apoorvpal01) [reply web/thesauruses/php/classes.json]: I have added a commit removing the code lines from features that are not implemented.

**Review by geekygirlsarah — APPROVED** [review]

**Comment by geekygirlsarah** [comment]
Thank you for the changes! I think it's ready for merging.

Don't forget to sign up for Hacktoberfest if you want this to count towards the shirt/swag!

**Pull request merged**